### PR TITLE
Feature/cleaning up code

### PR DESCRIPTION
**Pull request by jeffrey1200**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by jeffrey1200** [comment]
The app didn't break on my end once I deleted the files. If I deleted commented code that was needed, please save it!

File: README.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ You can find the deployed project at [https://www.expresswash.us/](https://www.e
...
 |        [<img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15">](https://www.linkedin.com/)        | [<img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15">](https://www.linkedin.com/in/daquan-freeman-a4b63622/) | [<img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15">](https://www.linkedin.com/in/ryan-ansberry-11a55415a/) | [<img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15">](https://www.linkedin.com/in/malarc01/) | [<img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15">](https://www.linkedin.com/in/calehaug/) | [<img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15">](https://www.linkedin.com/in/sam-malanchuk/) |
```
Comment (DentonWarnock) [inline README.md:15]: Thanks for updating the team, the readme still needs the information below this updated.

File: src/App.js [hunk 2]
```diff
@@ -52,7 +52,6 @@ function App(props) {
         <GlobalStyle />
         <Switch>
           <Route exact path="/" component={LandingPage} />
-          <Route path="/instantQuote" component={QuoteGenerator} />
```
Comment (DentonWarnock) [inline src/App.js:55]: Removing the QuoteGenerator component will really help lower our code smells

**Review by DentonWarnock — COMMENTED** [review]
Thanks for updating the team info on the readme and cleaning up some code. There is still more updating for the Readme and code cleanup needed.

**Review by DentonWarnock — APPROVED** [review]
It would be great if you could add our TL V to the readme contributors as well!

**Pull request merged**